manifests/charts/aperture-controller/values.yaml

```diff
@@ -542,6 +542,8 @@ controller:
 ##
 etcd:
   enabled: true
+  autoCompactionMode: periodic
+  autoCompactionRetention: 10m
```

Comment:
The description for this in etcd chart says this value should be in hour:
```
@param autoCompactionRetention Auto compaction retention for mvcc key value store in hour,
```
So, I think we can set default to 24 hour as we have used the same in our customer demo cluster where we have the wavepool generator 24x7 and we haven't faced the size issue after setting that.